WordPress-Core/ruleset.xml

```diff
@@ -393,7 +415,7 @@
 	</rule>
 
 	<!-- Rule: create_function() function, which internally performs an eval(),
-		 is deprecated in PHP 7.2. Both of these must not be used. -->
+		 is deprecated in PHP 7.2. ... these must not be used. -->
```

Comment:
This doesn't look right?

Reply:
Wrong PR for that comment and it would need another PR to change it.

All the same, this was just a way to try and quote only the relevant bit of the handbook paragraph in the ruleset, similar to how it was done for the section above it on `eval()` (line 388/410).
> `<!-- Rule: The eval() construct is very dangerous, and is impossible to secure. ... these must not be used. -->`

The original paragraph is:
> The eval() construct is very dangerous, and is impossible to secure. Additionally, the create_function() function, which internally performs an eval(), is deprecated in PHP 7.2. Both of these must not be used.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#clever-code

Reply:
As a side-note: I'd be happy for that text to be improved, but as it's nothing user-facing, I don't think it's a reason to delay the release.